docs/topics/proc_deploying-the-booster-using-launcher-oso.adoc

```diff
@@ -8,13 +8,14 @@
 //
 // Rationale: This procedure is identical in all deployments
 
-[id='deploying-the-booster-using-launcher-oso_{context}']
-= Deploying the booster using {name-launcher-oso}
+[id='deploying-the-example-using-launcher-oso_{context}']
+[id='deploying-the-example-using-launcher-oso_{context}']
+= Deploying the example using {name-launcher-oso}
```

Comment:
Here again, "example application" makes more sense since we talk about deploying it.